desc/compute/_stability.py

```diff
@@ -200,3 +200,165 @@ def _magnetic_well(params, transforms, profiles, data, **kwargs):
         / (data["V_r(r)"] * data["<B^2>"])
     )
     return data
+
+
+@register_compute_fun(
+        name="gds2",
+        label="\\mathrm{gds2}",
+        units="~",
+        units_long="None",
+        description="Fieldline bending term1",
+        dim=3,
+        params=[],
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "grad(alpha)",
+            ]
+        )
+def _gds2(params, transforms, profiles, data, **kwargs):
+    data["gds2"]= dot(data["grad(alpha)"], data["grad(alpha)"])
+
+
+@register_compute_fun(
+        name="gds21",
+        label="\\mathrm{gds21}",
+        units="~",
+        units_long="None",
+        description="Fieldline bending term2",
+        dim=3
+        params=[]
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "grad(alpha)",
+            "grad(rho)"
+            ]
+        )
+def _gds21(params, transforms, profiles, data, **kwargs):
```

Comment:
can move this to `_metric.py` and rename to `g^ra`

Reply:
Done.